next-sitemap.config.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+	siteUrl: process.env.SITE_URL || 'https://developer.hashicorp.com',
+	generateRobotsTxt: true, // (optional)
+	exclude: ['/swingset/*'],
```

Comment:
I think we should also exclude `/onboarding/`, right @kendallstrautman ?

Reply:
Noticed `/swingset` is still being included too.

Reply:
Updated the matcher to `/swingset*`